test/data/ansible/check_ostree.yaml

```diff
@@ -908,6 +908,30 @@
             failed_counter: "{{ failed_counter | int + 1 }}"
       when: firewall_feature == "true"
 
+    # case: check sos package
+    - name: Check SOS Package
+      block:
+        - name: Generate sos report
+          command: sos report --batch --tmp-dir ./ -k podman.logs=on > output_test.log
+          become: "true"
+          register: result_sos_report
+
+        - assert:
+            that:
+              - result_sos_report is succeeded
+            fail_msg: "No usable sos package present"
+            success_msg: "sos package present and usable"
+      always:
+        - set_fact:
+            total_counter: "{{ total_counter | int + 1 }}"
+      rescue:
+        - name: failed count + 1
+          set_fact:
+            failed_counter: "{{ failed_counter | int + 1 }}"
+      when: ((ansible_facts['distribution'] == 'RedHat' and ansible_facts ['distribution_version'] is version('8.8', '>=')) or (ansible_facts['distribution'] == 'RedHat' and ansible_facts ['distribution_version'] is version('9.2', '>=')) or (ansible_facts['distribution'] == 'CentOS')) and
```

Comment:
@mcattamoredhat Could you please use when condition like https://github.com/virt-s1/rhel-edge/blob/main/check-ostree.yaml#L596? Thanks.